src/pages/TermsAndConditionsPage.js

```diff
@@ -0,0 +1,7 @@
+import TermsAndConditionsComponent from "../components/TermsandConditions/TermsAndConditions";
+import React from "react";
+
+const TermsAndConditionsPage = () => (
+    <TermsAndConditionsComponent />
```

Comment:
No need to separate into `*Page` and `*Component`, as this is a pretty simple page/component, just put it directly here